app/src/main/java/com/dessalines/thumbkey/ui/components/settings/lookandfeel/LookAndFeelActivity.kt

```diff
@@ -750,6 +736,54 @@ fun LookAndFeelActivity(
                     onValueChange = { text = it },
                     label = { Text(stringResource(R.string.test_out_thumbkey)) },
                 )
+                if (confirmReset.value) {
```

Comment:
Move this entire block above, into the correct reset onClick. That onclick should open this dialog.

Reply:
Seems like I cannot move it inside the onClick as this results in an error: 
`@Composable invocations can only happen from the context of a @Composable function`.

Reply:
Ah. In that case, name the variable like `showConfirm...` and move it to the top level.